credits.html

```diff
@@ -39,8 +39,8 @@ <h3>Previous Core Developers</h3>
         <h2>Contributors</h2>
         <p>
           Also see the complete lists of
-          <a href="https://github.com/minetest/minetest/graphs/contributors">Minetest</a> and
           <a href="https://github.com/minetest/minetest_game/graphs/contributors">Minetest Game</a>
+          <a href="https://github.com/minetest/minetest/graphs/contributors">Luanti</a> and
```

Comment:
"Also see the complete lists of Minetest Game Luanti and contributors?"

Was the intent to mention MTG first so there is no confusion as to what "Luanti Game" might be? Please move the "and" then.

Reply:
Didn't intend to change the ordering of it and don't know why it happened. Moved the line back.